.github/workflows/ci.yml

```diff
@@ -13,7 +13,7 @@ concurrency:
   cancel-in-progress: true
 
 jobs:
-  ci:
+  linter:
```

Comment:
Why linter 🥲 ? we usually name the same as file name, I think it's better to leave ci

Reply:
So that file contains 2 jobs that are related to ci

Reply:
I can split into 2 ones: ci and ci-build?

Reply:
ok, got it, let's split them not to run it on every trigger